src/strategy-heat-capacitor.js

```diff
@@ -29,6 +29,39 @@ module.exports = function (RED) {
       if (validateInput(node, msg)) {
         // Using msg.payload.config to change specific properties
         if (msg.hasOwnProperty("payload")) {
+          if (msg.payload.hasOwnProperty("commands")) {
+            //Commands override input
+            if (node.hasOwnProperty("schedule")) {
+              //Do not execute if schedule is missing
+              if (msg.payload.hasOwnProperty("time")) {
+                node.dT = findTemp(msg.payload.time, node.schedule);
+              } else {
+                node.dT = findTemp(DateTime.now(), node.schedule);
+              }
+              node.T = node.setpoint + node.dT;
+              if (msg.payload.commands.hasOwnProperty("sendSchedule")) {
+                // Send output if schedule exists
+                if (node.hasOwnProperty("schedule") && msg.payload.commands.sendSchedule == true) {
+                  node.send([
+                    null,
+                    null,
+                    { payload: node.schedule },
+                    { payload: { setpoint_now: node.T, schedule: node.schedule.minimalSchedule } },
+                  ]);
+                }
+              }
+              if (msg.payload.commands.hasOwnProperty("sendOutput") && msg.payload.commands.sendOutput == true) {
+                // Send output if schedule exists
+                node.send([
+                  { payload: node.T, topic: "setpoint", time: node.schedule.time, version: version },
+                  { payload: node.dT, topic: "adjustment", time: node.schedule.time, version: version },
+                  null,
+                  null,
+                ]);
+              }
+            }
+            return;
```

Comment:
What if there is commands in the same message as config and/or priceData? Should that be allowed?

Reply:
I considered the same topic... I could not see a case where this functionality would be beneficial and I believe it should be generally avoided... That is, the configuration/inputs are handled by one set of logic, while the output scheduling is handled by another. It would also demand a bit more logic (do we only output the schedule if the "outputSchedule" command is included with prices?

In the end, I lean towards forcing the user to send commands/prices & configurations separately...